apps/nextra/pages/en/build/cli/use-cli/working-with-move-contracts/local-simulation-benchmarking-and-gas-profiling.mdx

```diff
@@ -0,0 +1,251 @@
+import { FileTree } from "nextra/components";
+import { Callout } from 'nextra/components';
+
+# Local Simulation, Benchmarking & Gas Profiling
+
+## Overview
+The previous tutorial demonstrates how you can deploy and interact with Move contracts using various CLI commands.
+
+By default, those commands send a transaction to the remote fullnode for simulation and execution.
+You can override this behavior and simulate the transaction locally, by appending the following command line options of your preference:
```

Comment:
nit: by appending  *one of the* command line options? they are all local and mutually exclusive, right?